index.html

```diff
@@ -200,6 +221,94 @@ <h3>Profesional Art <br>
             <button class="see-project-btn" type="button" autofocus>See Project </button>
           </article>
         </div>
+
+        <!-- new created desktop work cards part -->
+
+        <div class="desktop-cards">
+          <article class="desktop-card-1">
+            <button type="button" autofocus>See Project </button>
+          </article>
+          <article class="desktop-card-2 desk-card">
+            <div class="card-text">
+              <h3> Data Dashboard <br>
+                Healthcare
+              </h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's
+                standard
+              </p>
+              <ul>
+                <li>Html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
```

Comment:
- [ ] As stated 😃 by the previous reviewer try to add the hover effect at each card I see that you put the style of hover at one card and it's static not hover, so kindly 😊try to add the hover dynamically when the visitor hovers the card the see project button or link will appear to the visitor 💯

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/214830155-5093c7fa-2072-4148-b305-734a11c00a38.png)|   ![image](https://user-images.githubusercontent.com/83360636/214830265-be0c9ff2-9af0-4690-8bcd-a7a8b7d48f7b.png)|